src/models/Popup.js

```diff
@@ -452,6 +523,10 @@ export default class Popup extends Base {
     // returns the parent element with the specified class, or the original element - null if not found
     const btn = traverseDOMPath(event.target, 'cc-btn') || event.target
     
+    if (btn.classList.contains( 'cc-btn' ) && btn.classList.contains( 'cc-customize' )){
```

Comment:
Consistent formatting please:

```suggestion
    if (btn.classList.contains('cc-btn') && btn.classList.contains('cc-customize')) {
```

Reply:
formatting is mixed as there is no prettier introduced yet - will be done later